### ISSUE TITLE
Navigation Bar don't change color on white background

### ISSUE THREAD

**Issue by libnijunior** [libnijunior/patchrom_bullhead#7]

**Comment by Matoxxi** [comment]
that's due to miui framework you have this bug even on official miui,atm it works only on miui apps

**Comment by libnijunior** [comment]
Yes, it is an bug present on miui patchrom, but I will try to get code from nexus 5 MIUI 8, that don't have this glitch...
